src/config/dynamic-theme-fixes.config

```diff
@@ -9154,6 +9154,39 @@ CSS
 
 ================================
 
+my.contabo.com
+
+CSS
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
```

Comment:
Cosmetic change?

Reply:
this tries to match the original, because it replaced the image that was there befor it.